docs/config.toml

```diff
@@ -36,6 +36,7 @@ home = [ "HTML", "RSS", "SearchIndex" ]
     { name = "Releases", pre = "relative", url = "../../releases", weight = 600 },
     { name = "Blogs", pre = "relative", url = "../../blogs", weight = 998 },
     { name = "Talks", pre = "relative", url = "../../talks", weight = 999 },
+    { name = "Vendors", url = "/vendors", weight = 1000 },
```

Comment:
nit: maybe stick to the `weight` order - if I understand correctly `weight` defines the order of the menu items, and while it is valid to have them on different order than the point are enumerated this is harder to read.

Reply:
Fixed